W.A.J. P to demonstrate try catch block, 










W.A.J. P to demonstrate multiple catch blocks, (one is to handle divide by zero 
exception and another one is to handle 
ArrayIndexOutOfBoundException) int a [] =new int [5]; a [5]=30/0;

ans:
public class Multitrycatch
{
    public static void main(String[] args) 
    {
        int a[ ] = new int[5];

        try 
        {
            a[5] = 30 / 0; 
        } 
        catch (ArithmeticException e) 
        {
            System.out.println("Division by zero is not allowed.");
        } 
        catch (ArrayIndexOutOfBoundsException e) 
        {
            System.out.println("use only 5 no");
        }
    }
}
 